apiserver/appcat/update.go

```diff
@@ -0,0 +1,15 @@
+package appcat
+
+import (
+	"context"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+)
+
+var _ rest.Updater = &appcatStorage{}
+var _ rest.CreaterUpdater = &appcatStorage{}
+
+func (s appcatStorage) Update(ctx context.Context, name string, objInfo rest.UpdatedObjectInfo, createValidation rest.ValidateObjectFunc, updateValidation rest.ValidateObjectUpdateFunc, forceAllowCreate bool, options *metav1.UpdateOptions) (runtime.Object, bool, error) {
+	return nil, false, nil
```

Comment:
Nitpick: This will result in a weird error when someone tries to update an appcat listing.